apps/docs/components/Navigation/Footer.tsx

```diff
@@ -1,15 +1,107 @@
 import Link from 'next/link'
 import { Button } from 'ui'
+import { useIsLoggedIn, useIsUserLoading } from 'common'
 import { primaryLinks, secondaryLinks } from '~/data/footer'
 import { LayoutMainContent } from '~/layouts/DefaultLayout'
 
-const Footer = () => (
-  <LayoutMainContent className="pt-0">
-    <div className="mt-16">
-      <ul className="flex flex-col gap-2">
-        {primaryLinks.map(({ url, featherIcon: Icon, icon, text, ctaLabel }) => (
-          <li key={url} className="flex items-center gap-1 text-xs text-foreground-lighter">
-            {icon && (
+const Footer = () => {
+  const isLoggedIn = useIsLoggedIn()
+  const isUserLoading = useIsUserLoading()
+
+  return (
+    <LayoutMainContent className="pt-0">
+      <div className="mt-16">
+        {!isUserLoading && (
+          <Button asChild className="mb-8">
+            <a href="https://supabase.com/dashboard" target="_blank" rel="noreferrer noopener">
+              {isLoggedIn ? 'Go to Dashboard' : 'Sign up for Supabase'}
+            </a>
+          </Button>
+        )}
+        <ul className="flex flex-col gap-2">
```

Comment:
The giant diff here is being triggered by the indentation shift, caused by going from implicit return to explicit return statement in this arrow function. It'll be easier to review if you turn on "Hide whitespace".